apps/web/next.config.js

```diff
@@ -15,6 +15,7 @@ const withTM = require("next-transpile-modules")([
   "@calcom/prisma",
   "@calcom/trpc",
   "@calcom/ui",
+  "@calcom/console",
```

Comment:
This is to be able to import useLocale from `console` which in turn uses the one from web